TestPullRequest/src/testPackage/Calc.java

```diff
@@ -0,0 +1,40 @@
+package testPackage;
+
+public class Calc extends SuperCalc implements CalcInterface {
+	
+	public final static int MULTIPLIER = 10;
+
+	public Integer addition(int a, int b, int c) {
+		return a - b;
+	}
+	
+	public Integer subtraction(int a, int b, int c) {
+		return a - b;
+	}
+	
+	public Integer subtraction(int a, int b) {
+		return a - b;
+	}
+	
+	public Integer multiplication(int a, int b, int c) {
+		return a * b;
+	}
+
+	@Override
+	public void testInterface2() {
```

Comment:
@LHommeDeBot: Can you please rename this method to something like 'newName'???? Contact me on 'https://www.example.com/foo/?bar=baz&inga=42&quux'